content/2016/09/30/cssconf.rst

```diff
@@ -0,0 +1,608 @@
+public: yes
+author: miriam
+tags: [conference, front-end]
+.. image:
+..   - src: 'cssgriddemo.jpg'
+summary: |
+  It was a pleasure speaking at
+  CSSConf 2016 in Boston!
+  Here are some of my notes from the event.
```

Comment:
Can you add more teaser info in the summary about what's in the blog post since there's such good stuff in there?